### PR TITLE
add manual trigger for build

### PR DESCRIPTION
**Pull request by kmdalton**
I expect this to fail because numpy version 2 will break our CI

### CODE REVIEW

**Review by JBGreisman — APPROVED** [review]
This will be useful -- I'll merge it in after we decide what order to merge the remaining PRs in.

**Comment by JBGreisman** [comment]
This should all be fine, but I think it just requires a rebase/fast-forward to include latest changes

**Comment by kmdalton** [comment]
@JBGreisman, I think this is good to go. I already merged the other PR.

**Pull request merged**
